### DISCUSSION TITLE
progress while downloading

### DISCUSSION THREAD

**Discussion by robert-rozee** [FreeTubeApp/FreeTube#4269] [Q&A]
hi,
    is there any way to see the progress of a download? i was just trying to download a clip that was over an hour of video and near 500mb in length. once i'd selected to download it, there was no way i could find to check that the video was actually downloading. it did eventually appear in the download location, but this took 5 minutes or so. not a problem with short clips, but an issue for new users with anything that is very long.

cheers,
rob   :-)

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Hi @robert-rozee download progress is not available and will get implemented with the implementation of the download tab https://github.com/FreeTubeApp/FreeTube/issues/1772

**Reply by robert-rozee** [reply]
much appreciated - btw, this aside the program worked perfectly for what i needed.

cheers,
rob   :-)
